### ISSUE TITLE
Unable to install AMD GPU Plugin

### ISSUE THREAD

**Issue by mjbneps** [Rem0o/FanControl.Releases#1923]
My fan controls download didn't have a "Plugins" folder so I made one. I've tried the "Release.zip" files from AppVeyor and/or from the "Code > Download Zip" here: https://github.com/Rem0o/FanControl.ADLX. I've extracted all the files into the plugin folder, unblocked them all; tried extracting only the Dlls files themselves; tried extracting all files and only unblocking the Dlls files. Fan Control either does not recognize the plugin or fails to launch completely. What am I doing wrong here? 

https://imgur.com/a/d0IANhR

**Comment by mjbneps** [comment]
FWIW, I was able to upload this plugin a few days ago and had it working, but have been trying to diagnose a separate issue and reinstalled fan control. I do not know how I did anything differently that made it work the first time. I'm at a loss right now.

**Comment by mjbneps** [comment]
I was able to get it working again, but now whenever I boot up the pc it does not default to my new config; my case fans throttle to 100% until I manually close fan control and re-open it. Same occurs when my pc goes to sleep mode.

**Comment by Rem0o** [comment]
Since this issue is related to the ADLX plugin, please move your issue to that repo.
